Modules/Core/Common/include/itkMath.h

```diff
@@ -516,9 +516,9 @@ template <typename TInputType1, typename TInputType2>
 struct AlmostEqualsScalarImplementer
 {
   static constexpr bool TInputType1IsInteger = std::is_integral<TInputType1>::value;
-  static constexpr bool TInputType1IsSigned = itk::NumericTraits<TInputType1>::IsSigned;
+  static constexpr bool TInputType1IsSigned = std::is_signed<TInputType1>::value;
```

Comment:
In commit message, `IsInteger` should be `IsSigned`. e.g. `STYLE: Remove IsInteger and replace with std::is_signed` -> `STYLE: Remove NumericTraits::IsSigned and replace with std::is_signed`.